### DISCUSSION TITLE
🛫 Version 1.1.0

### DISCUSSION THREAD

**Discussion by valnoxy** [valnoxy/t480-oc#4] [Announcements]
Hey 👋,
I hope you all have a great day!

# Changes
The following macOS versions/variants have been added:
   - HeliPort
   - macOS 13 Mammoth (pre-release)

The following kexts / bootloaders have been updated:
   - AppleALC to ```1.7.1```
   - OpenCore to ```0.8.0```

<hr /><em>This discussion was created from the release <a href='https://github.com/valnoxy/t480-oc/releases/tag/v1.1.0'>🛫 Version 1.1.0</a>.</em>

**Comment by askwakhid** [comment]
Hi,
Your EFI works fine on my Thinkpad T480s Monterey 11.2.1.
Thank you very much.
